AnnoDesigner/AnnoCanvas.xaml.cs

```diff
@@ -1291,44 +1262,48 @@ protected override void OnMouseUp(MouseButtonEventArgs e)
                 switch (CurrentMode)
                 {
                     case MouseMode.Standard:
-                        if (CurrentObjects.Count == 0)
                         {
-                            var obj = GetObjectAt(_mousePosition);
-                            if (obj == null)
+                            if (CurrentObjects.Count == 0)
                             {
-                                if (!(IsControlPressed() || IsShiftPressed()))
+                                var obj = GetObjectAt(_mousePosition);
+                                if (obj == null)
                                 {
-                                    // clear selection
-                                    _selectedObjects.Clear();
+                                    if (!(IsControlPressed() || IsShiftPressed()))
+                                    {
+                                        // clear selection
+                                        _selectedObjects.Clear();
+                                    }
+                                }
+                                else
+                                {
+                                    // remove clicked object
```

Comment:
might `selected` being a better term rather than `clicked`?

Reply:
I changed it.